_data/simple-icons.json

```
@@ -14146,6 +14146,11 @@
             "hex": "0E3A2F",
             "source": "https://www.skoda-connect.com"
         },
+        {
+            "title": "sktime",
+            "hex": "0078BD",
```

`#3fb1c5` is the `--pst-color-primary` variable, which is the color used on links and buttons. I'd go with this as the primary color, and I'm guessing it's one of the colors in the original logo.
```suggestion
            "hex": "3FB1C5",
```

---

Actually, this looks like FFR's color, [not sktime's](https://github.com/sktime/sktime/blob/51f5b46905dc178bbddc21e9309e502c0a76152c/docs/source/images/sktime-logo-no-text.svg?short_path=a9845a8#L211). Should I change it to this color reference in my PR instead? Thanks!

---

Definitely sktime's color ahah but maybe a newer one that they don't use in the logo. Sure, change it to `#0177B9`.